blocks/cards/cards.css

```diff
@@ -142,7 +145,10 @@
       padding: 80px;
   }
 
-  .cards.profile-summary .cards-card-body h1,h2,h3,h4 {
+  .cards.profile-summary .cards-card-body h1,
+  .cards.profile-summary .cards-card-body h2,
```

Comment:
why all headings has same size ?

Reply:
Not sure, but earlier css was written like .cards.profile-summary .cards-card-body h1,h2,h3,h4 {}. As a result h2,h3,h4 styling got impacted globally as the css is not localized to the cards block. I just made it localized for now.